cmd/metrics-v3-types.go

```diff
@@ -232,10 +232,13 @@ func (m *MetricValues) Set(name MetricName, value float64, labels ...string) {
 	if !ok {
 		v = make([]metricValue, 0, 1)
 	}
-	m.values[name] = append(v, metricValue{
-		Labels: labelMap,
-		Value:  value,
-	})
+	// If valid non zero value set the metrics
+	if value > 0 {
```

Comment:
What if it is less than 0?

Reply:
All counters and gauges have no negative values in our system AFAIK.

Reply:
For the time being this is fine, if we get negative values we can add relevant checks subsequently.

We don't have to add code for future metrics.